custom/abt/reports/expressions.py

```diff
@@ -113,9 +113,9 @@ def _get_language(cls, item):
 
     @classmethod
     def _get_warning(cls, spec, item):
-        default = spec.get("warning", "")
+        default = unicode(spec.get("warning", u""))
         if cls._get_language(item) == "fra":
-            return spec.get("french", default)
+            return unicode(spec.get("french", default))
```

Comment:
Yeah, unfortunately there isn't a flag you can set or anything, but there is this: http://stackoverflow.com/a/2967461/1739725